### DISCUSSION TITLE
Can mapsui.maui display a satellite view?

### DISCUSSION THREAD

**Discussion by upswing1** [Mapsui/Mapsui#1838] [Q&A]

**Comment by pauldendulk** [comment]
@upswing1 Mapsui is not tied to any specific kind of data. We use the openstreetmap default map in many of our samples but other data sources can be configured as well.

The BingSample shows use of bing aerial imagery. This is however a service you need to pay for. You need to acquire an api key from Microsoft:
https://github.com/Mapsui/Mapsui/blob/ee45adbeeb83ad553228a69f29479cd56816eff1/Samples/Mapsui.Samples.Common/Maps/Demo/BingSample.cs#L25

You can see the map if you download the Mapsui source, run one of the sample apps, like Mapsui.Samples.Avalonia, and select the sample called "3 Virtual Earth" in the "Demo" category.
